lighthouse-core/lib/network-recorder.js

```diff
@@ -303,15 +303,22 @@ class NetworkRecorder extends EventEmitter {
     request.onResourceChangedPriority(data);
   }
 
+  /**
+   * @param {LH.Crdp.Target.ReceivedMessageFromTargetEvent} data
+   */
+  onReceivedMessageFromTarget(data) {
+    /** @type {LH.Protocol.RawMessage} */
+    const protocolMessage = JSON.parse(data.message);
+    if ('id' in protocolMessage) return;
```

Comment:
and a comment here that (I'm assuming like in `connection.js`) if it has an `id` it's a command response, and we only care about events

Reply:
done